src/governance-vote.h

```diff
@@ -68,6 +68,8 @@ enum vote_signal_enum_t  {
     VOTE_SIGNAL_CUSTOM20   = 35
 };
 
+static const int MAX_SUPPORTED_VOTE_SIGNAL = 4;
```

Comment:
Well, I'm not sure where this assumption comes from in the first place tbh and I see no issues using smth that is defined in language specification. Anyway, I'm kind of ok with ef6aa2e too.

>I can't think of any reason we would want to reassign the first 4 values.

Yeah, that was more like a general example for enums, not for this specific case of course :)